tools/cldr-apps/js/src/views/AnnounceForm.vue

```diff
@@ -166,12 +171,14 @@ export default defineComponent({
     },
 
     describeLocs() {
+      if (this.formState.locs === "!") return "Non-TC locales";
       return this.formState.locs === "" || this.formState.locs === "*"
         ? "all locales"
         : "the following locale(s): " + this.formState.locs;
     },
 
     validateLocales() {
+      if (this.formState.locs === "!") return;
```

Comment:
previously any string, including "*", got passed to the back end for validation and normalization. This exception for "!" skips the validation. Not sure whether that's a problem... I guess not... on the back end, combineRegionalVariants doesn't check for permission...

Reply:
A defined constant like cldrAnnounce.LOCS_NON_TC (similar to the back end) could make this "!" more robust

Reply:
In the current PR the "!" is entirely on the back end.